src/components/Contacts/index.js

```diff
@@ -30,16 +36,13 @@ function Contacts() {
 
   function emailHandler(e) {
     setEmail(e.target.value);
-    // console.warn(e.target.value)
   }
 
   function msgHandler(e) {
     setMessage(e.target.value);
-    // console.warn(e.target.value)
   }
   function subjectHandler(e) {
     setSubject(e.target.value);
-    // console.warn(e.target.value)
   }
 
   function sendEmail(e) {
```

Comment:
You can shorten these errors quite a bit...
```
  if (name.length < 4) {
      setNameErr(true);
    } else {
      setNameErr(false);
    }
 ```
 is the same as
 ```
     setNameErr(name.length < 4);
 ```
 
 You can do that with each of your fields.